src/components/LocationFilter.jsx

```diff
@@ -1,40 +1,40 @@
-import * as React from "react"
 import Fuse from "fuse.js"
-import Link from "next/link"
 import { useRouter } from "next/router"
 import FilterButton from "./FilterButton"
-import { useState } from "react"
+import { useEffect, useState } from "react"
 import LocationIcon from "../assets/Location.svg"
 import SearchIcon from "../assets/Search.svg"
 import { HiChevronDown as DownArrow } from "react-icons/hi"
 import { HiChevronUp as UpArrow } from "react-icons/hi"
-import { useSlug } from "~/context/slug"
-import { useData } from "~/context/data"
+import { useCities } from "~/hooks/useCities"
+import Skeleton from "react-loading-skeleton"
+
+export default function LocationFilter({ city, resource }) {
+  const { data, error } = useCities()
+  const [resources, setResources] = useState(data)
+  const [cityState, setCityState] = useState(city ? true : false)
+
+  useEffect(() => {
```

Comment:
Move this useEffect hook to be inside useCities. Use cities should return an Array if that is what the component consumes.